docs/examples/alb-ingress-controller.yaml

```diff
@@ -12,42 +12,37 @@ metadata:
   # namespace scope, see --watch-namespace.
   namespace: kube-system
 spec:
-  replicas: 1
   selector:
     matchLabels:
       app: alb-ingress-controller
-  strategy:
-    rollingUpdate:
-      maxSurge: 1
-      maxUnavailable: 1
-    type: RollingUpdate
   template:
     metadata:
-      creationTimestamp: null
       labels:
         app: alb-ingress-controller
     spec:
       containers:
-        - args:
+        - name: alb-ingress-controller
+          # args:
             # Limit the namespace where this ALB Ingress Controller deployment will
             # resolve ingress resources. If left commented, all namespaces are used.
             # - --watch-namespace=your-k8s-namespace
 
             # Setting the ingress-class flag below ensures that only ingress resources with the
             # annotation kubernetes.io/ingress.class: "alb" are respected by the controller. You may
             # choose any class you'd like for this controller to respect.
-            - --ingress-class=alb
+            # - --ingress-class=alb
```

Comment:
i think this should remain uncommented.
The behavior of "ingress-class" parameter is slightly tweaked since v1.0.0. If it's left unspecified, the controller will target ingress `without ingress.class` or alb. It's better to remain restricted to `alb` for example setup.

Reply:
Heh, that happens to be the one thing I was on the fence about. No problem!